### DISCUSSION TITLE
Better Fullscreen Camera Options

### DISCUSSION THREAD

**Discussion by tophattwaffle** [OctoFarm/OctoFarm#765] [Feature Request]
Add ability for fullscreen to maximize the camera to the size of the browser window. This could be a toggle switch in settings to keep the existing implementation and the new fullscreen.

The current fullscreen implementation also is bad for displays that don't match the camera's aspect ratio.

Currently it works like this:
![image](https://user-images.githubusercontent.com/6774125/137618616-35f6c220-c304-4e66-a11a-df0e8c919e8d.png)

Ideally this would be how the new setting works:
![2021-10-17_03-28-26](https://user-images.githubusercontent.com/6774125/137618636-445ef452-b8fb-4f0e-a417-5c192b342693.jpg)

**Comment by NotExpectedYet** [comment]
I haven't added a temp overlay but the next version will allow this. Just trying to center the camera. 

Will add a details overlay in another version

It's activated with a single click. The old method remains with the double click. 

![image](https://user-images.githubusercontent.com/61137988/193439458-d09e7883-3af1-4c41-9264-a7e3208fc499.png)
